locations/jclouds/src/main/java/org/apache/brooklyn/location/jclouds/JcloudsLocation.java

```diff
@@ -1510,7 +1510,7 @@ public Template buildTemplate(ComputeService computeService, ConfigBag config) {
             if (image==null) throw new NullPointerException("Template does not contain an image (templateBuilder.build returned invalid template)");
         } catch (AuthorizationException e) {
             LOG.warn("Error resolving template: not authorized (rethrowing: "+e+")");
```

Comment:
Would prefer that something (e.g. this log.warn message) says what actually failed (i.e. includes the `templateBuilder`, or some other appropriate description). Otherwise it can be very hard to figure out from a customer's logs what failed and why, when there are multiple apps being deployed at the same time.

Agree that removing it from the propagated exception makes sense.